schemas/test/en/test_list_collector_repeating_blocks.json

```diff
@@ -0,0 +1,227 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Test a Simple List Collector with Repeating Blocks",
+    "theme": "default",
+    "description": "A questionnaire to test a simple list collector with repeating blocks and no additional blocks or sections.",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "answer_codes": [
+        {
+            "answer_id": "list-collector-next-question-answer-1",
+            "code": "1"
+        },
+        {
+            "answer_id": "list-collector-add-block-question-answer",
+            "code": "1a"
+        },
+        {
+            "answer_id": "repeating-block-1-question-answer-1",
+            "code": "1b"
+        },
+        {
+            "answer_id": "repeating-block-1-question-answer-2",
+            "code": "1c"
+        },
+        {
+            "answer_id": "repeating-block-2-question-answer-1",
+            "code": "1d"
+        },
+        {
+            "answer_id": "repeating-block-2-question-answer-2",
+            "code": "1e"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Linear",
+        "options": {
+            "summary": {
+                "collapsible": false
+            }
+        }
+    },
+    "sections": [
+        {
+            "id": "section",
+            "groups": [
+                {
+                    "id": "group",
+                    "title": "List",
+                    "blocks": [
+                        {
+                            "id": "list-collector-block",
+                            "type": "ListCollector",
+                            "for_list": "repeating_blocks_list_collector",
+                            "question": {
+                                "id": "list-collector-next-question",
+                                "type": "General",
+                                "title": "Do you need to add any more to this list?",
```

Comment:
As per new rule - I think context of the schema could be improved (subject to not creating too much overhaul in tests)

Reply:
Will augment title on list collector and repeating blocks